README.md

```diff
@@ -124,12 +118,44 @@ For in-depth examples of NNCF integration, browse the [sample scripts](#model-co
 For FAQ, visit this [link](./docs/FAQ.md).
 
 
+## Usage of Post-Training Quantization
+
+The basic workflow steps:
+1) Initialize the dataset transformation function.
+2) Initialize NNCF Dataset.
+3) Run the quantization pipeline.
+
+NNCF provides [samples](#post-training-quantization) that demonstrate Post-Training Quantization usage for PyTorch, TensorFlow, ONNX, OpenVINO.
+
+```python
+import onnx
+import nncf
+import torch
+import torchvision.datasets as datasets
+
+# Instantiate your uncompressed model
+onnx_model = onnx.load_model('/model_path')
+# Provide validation part of the dataset for statistics collection for compression algorithm
+representative_dataset = datasets.ImageFolder("/path")
+dataset_loader = torch.utils.data.DataLoader(representative_dataset, batch_size=1)
+# Step 1: Initialize transformation function
+input_name = onnx_model.graph.input[0].name
+def transform_fn(data_item):
+    images, _ = data_item
+    return {input_name: images.numpy()}
```

Comment:
Shouldn't we note, that the output of the `transform_fn` should look as the model awaits?

Reply:
I put special note about the transformation function. Please take a look

Reply:
ONNX model can be executed with the input only as `Dict`?

Reply:
> ONNX model can be executed with the input only as `Dict`?

In the user API - yes. The way using AC is only for benchmarking purpose.